frontend/src/app/utils/smartyStreets.ts

```diff
@@ -1,23 +1,13 @@
-import { core, usStreet } from "smartystreets-javascript-sdk";
+import { usStreet, usZipcode } from "smartystreets-javascript-sdk";
 
+import { getZipCodeClient, getClient } from "./smartyStreetsClients";
 import { toLowerStripWhitespace } from "./text";
 
-class SmartyStreetsError extends Error {
-  constructor(message: string) {
-    super(message);
-    this.name = "SmartyStreetsError";
-  }
-}
-
-const getClient = () => {
-  if (process.env.REACT_APP_SMARTY_STREETS_KEY === undefined) {
-    throw new SmartyStreetsError("Missing REACT_APP_SMARTY_STREETS_KEY");
-  }
-  const credentials = new core.SharedCredentials(
-    process.env.REACT_APP_SMARTY_STREETS_KEY
-  );
-  return core.buildClient.usStreet(credentials);
-};
+// cf. https://github.com/DefinitelyTyped/DefinitelyTyped/blob/11436c5a19fc6aabfd6b8d93b37dac38b4ab9bc2/types/smartystreets-javascript-sdk/index.d.ts#L625
```

Comment:
what does the comment do?

Reply:
Ah, looks like I deleted my helpful comment explaining it. Basically, the provided `usZipcode.Result` type stubbornly insists that `status` and `reason` should be present, but in fact those properties only appear on responses for invalid ZIP codes.